test/InterchainTokenService.js

```diff
@@ -11,13 +11,16 @@ const {
 } = ethers;
 const { getCreate3Address } = require('@axelar-network/axelar-gmp-sdk-solidity');
 const { approveContractCall, approveContractCallWithMint } = require('../scripts/utils');
-const { getRandomBytes32, getRandomInt, expectRevert, gasReporter, getEVMVersion } = require('./utils');
-const { deployAll, deployContract, deployInterchainTokenService } = require('../scripts/deploy');
+const { getRandomBytes32, getRandomInt, expectRevert, gasReporter, getEVMVersion, itsHubChainName } = require('./utils');
+const { deployAll, deployContract, deployInterchainTokenService, deployAllWithTestGatewayCaller } = require('../scripts/deploy');
 
 const MESSAGE_TYPE_INTERCHAIN_TRANSFER = 0;
 const MESSAGE_TYPE_DEPLOY_INTERCHAIN_TOKEN = 1;
 const MESSAGE_TYPE_DEPLOY_TOKEN_MANAGER = 2;
-const INVALID_MESSAGE_TYPE = 3;
+/* eslint-disable-next-line no-unused-vars */
+const MESSAGE_TYPE_SEND_TO_HUB = 3;
```

Comment:
Note: This part needs to be refactored. e.g. move const variables to `constants.js`